sdk/trace/provider.go

```diff
@@ -367,6 +367,7 @@ func ensureValidTracerProviderConfig(cfg tracerProviderConfig) tracerProviderCon
 		cfg.idGenerator = defaultIDGenerator()
 	}
 	cfg.spanLimits.ensureDefault()
+	cfg.spanLimits.parsePotentialEnvConfigs()
```

Comment:
Should this happen before 'ensureDefault()` is invoked to ensure that invalid values from the environment are set to the default?